pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  
+  <groupId>org.dataconservancy.pass</groupId>
+  <artifactId>pass-client</artifactId>
+  <packaging>pom</packaging>
+  <version>0.0.1-SNAPSHOT</version>
+  
+  <name>PASS Client Tool</name>
+  
+  <modules>
+    <module>pass-model</module>
+    <module>pass-json-adapter</module>
+    <module>pass-fedora-client</module>
+    <module>pass-client-api</module>
+    <module>pass-model-ext-nihms</module>
+    <module>pass-client-util</module>
+  </modules>
+  
+  <properties>
+  	<maven-compiler-plugin.version>3.6.1</maven-compiler-plugin.version>
+  	<maven-surefire-plugin.version>2.20.1</maven-surefire-plugin.version>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <joda-time.version>2.9.9</joda-time.version>
+    <org-json.version>20180130</org-json.version>
+    <junit.version>4.12</junit.version>
+    <jackson.version>2.9.4</jackson.version>
+    <fcrepo-java-client.version>0.3.0</fcrepo-java-client.version>
+    <logback.version>1.2.3</logback.version>
+  </properties>
+
+
+  <build>    
+    <plugins>
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>${maven-compiler-plugin.version}</version>
+        <configuration>
+          <source>1.8</source>
+          <target>1.8</target>
+        </configuration>
+      </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>${maven-surefire-plugin.version}</version>
+      </plugin>
+      
+    </plugins>
+  </build>
+
+  <dependencies>
```

Comment:
I would suggest use of `<dependencyManagment>` for managing versions of dependencies and `<exclusions>` used for all modules in this project.  Keeping a `<dependencies>` element for common dependencies makes sense as long as it doesn't pollute the runtime or compile time classpaths (e.g. adding a dependency to the parent pom with `compile` or `runtime` scope that isn't used by one of the modules)

Reply:
Modules, then, should not need to specify a `<version>` or `<exclusions>` element for plugins or dependencies; they will be obtained from the parent.  This insures all modules are using the same versions and exclusions.